packages/autorest.python/autorest/codegen/models/model_type.py

```diff
@@ -308,13 +312,17 @@ def imports(self, **kwargs: Any) -> FileImport:
 class GeneratedModelType(ModelType):  # pylint: disable=abstract-method
     def type_annotation(self, **kwargs: Any) -> str:
         is_operation_file = kwargs.pop("is_operation_file", False)
-        retval = f"_models.{self.name}"
-        if self.internal:
-            retval = f"{self.code_model.models_filename}.{retval}"
-        return retval if is_operation_file else f'"{retval}"'
+        skip_quote = kwargs.get("skip_quote", False)
+        retval = typing_name(
```

Comment:
i think it's messier to , because looking at this code, it's a bunch of information that goes into a black box with .

I would be ok with either duplicating this code, or having a helper function that is much simpler, like `add_models_file_to_path`

Reply:
Fine. To make it clear, I remove the helper function